### DISCUSSION TITLE
Password protaction Vercel

### DISCUSSION THREAD

**Discussion by gilipsy** [oleeskild/obsidian-digital-garden#314] [Q&A]
Is there any way to set a password for my garden?

**Comment by gilipsy** [comment]
https://github.com/vercel/examples/tree/main/edge-middleware/basic-auth-password
I found this, but I don't understand how to connect it to my project and if it can work together with the obsidian digital garden

**Comment by lehoangphuc747** [comment]
I also want to create a password for a website or a certain node but don't know how.
